doc/project2-RaftKV.md

```diff
@@ -202,6 +202,8 @@ In this stage, you may consider these errors, and others will be processed in pr
 > - Record the callback of the command when proposing, and return the callback after applying.
 > - For the snap command response, should set badger Txn to callback explicitly.
 > - After 2A, some tests you may need to run them multiple times to find bugs
+> - Use the log package to help debug code that you think may be problematic
```

Comment:
That's what the hint `Set the environment variable LOG_LEVEL=debug which may help you in debugging, see also the all available log levels.` in Part B indicates

Reply:
Thanks

Reply:
Thanks your review!